### DISCUSSION TITLE
Forward and back buttons for built-in browser or pressing Esc should go back one level at a time.

### DISCUSSION THREAD

**Discussion by FazlyMR** [ppy/osu#22359] [Ideas] [answered]
Once selecting the user's profile to view their profile page, then selecting a beatmap, if the user wants to go back to their profile page then the obvious way to go back is to press the Escape (Esc) button but this returns the user directly to the main menu instead of one level back to the user profile page.

[browser.webm](https://user-images.githubusercontent.com/69378798/213934769-a454ec16-763a-401c-8c26-67aff3d9fe54.webm)

Maybe a forward and back button that exists in all web browsers could be useful, or that pressing the Esc button should allow the user to go back one level at a time. If a user instead wants to go back to the main menu, having them move their cursor to the edge outside of the page and pressing any mouse button should then return them to the main menu.

![Screenshot from 2023-01-23 04-10-00](https://user-images.githubusercontent.com/69378798/213935455-eba050b5-a0c2-4420-a3d3-4d687cf2bf46.png)

**Comment by Theighlin** [comment, marked as answer]
Duplicate of #5063
